athletics.lic

```diff
@@ -66,6 +66,15 @@ class Athletics
     stow_athletics_items
   end
 
+  def outdoorsmanship_waiting(n)
+    start = Time.now
```

Comment:
Since this script is about athletics training, some folks (👋) might not realize why outdoorsmanship gets called. Suggesting we put an echo to clarify.

```suggestion
    echo "Need to wait a bit before can revisit athletics training obstacles"
    start = Time.now
```

Reply:
I'm trying to remove bloat, not add more.  99% of the people who play DR know that there's a timer for learning between climbs.

Reply:
> I'm trying to remove bloat, not add more

Hi Binu, when you use those words to describe my feedback it hurts my feelings and demotivates me to contribute. 💔 I'm not trying to create problems. Like you, I'm trying to support and improve people's experience with Lich.

I respect your perspective on the echos. That's fine. Many of you have been playing this game and using Lich far longer than I have. My intentions behind the suggestions was only to help coach/teach those who use or maintain the scripts to know a little bit more about why it was doing what it does. 👨‍🏫

Thanks

Reply:
I'm not trying to be rude at all here.  Let me explain it like this.

Those messages would be useful the 1st time someone sees them and then unnecessary scroll anytime after.  It's one of those things that's better to let people ask on lnet or the discord server and learn that way, rather than to have it messaging every time.

The outdoorsmanship training has been in the script for a long time.  Maybe I'll add a comment in the script itself as to what it's doing.

I appreciate your contributions to lich and I'm not trying to discourage you from continuing.  I was busy and so I left a short comment in response to your suggestions so I could resolve them and let the script get merged.  It wasn't meant to be rude.

Reply:
> I'm not trying to be rude at all here. Let me explain it like this.

Thank you for the follow up and clarification. 🤗 

> Those messages would be useful the 1st time someone sees them and then unnecessary scroll anytime after.

Excellent point. I feel super silly suggesting it now. 🐑 

> Maybe I'll add a comment in the script itself as to what it's doing.

😄 

> I appreciate your contributions to lich and I'm not trying to discourage you from continuing. I was busy and so I left a short comment in response to your suggestions so I could resolve them and let the script get merged. It wasn't meant to be rude.

❤️ Thanks, Binu